<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mesa</class>
 <widget class="QDialog" name="Mesa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="C1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>C1</string>
   </property>
  </widget>
  <widget class="QLabel" name="C2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>C2</string>
   </property>
  </widget>
  <widget class="QLabel" name="C3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>59</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>C3</string>
   </property>
  </widget>
  <widget class="QLabel" name="C4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>59</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>C4</string>
   </property>
  </widget>
  <widget class="QLabel" name="C5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>C5</string>
   </property>
  </widget>
  <widget class="QLabel" name="C6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>59</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>C6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Iniciar</string>
   </property>
  </widget>
  <widget class="QLabel" name="C7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>203</y>
     <width>59</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>C7</string>
   </property>
  </widget>
  <widget class="QLabel" name="C8">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>209</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>C8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P8">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P8</string>
   </property>
  </widget>
  <widget class="QLabel" name="Nombre">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Jugador</string>
   </property>
  </widget>
  <widget class="QLabel" name="Puntos1">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Puntos1</string>
   </property>
  </widget>
  <widget class="QLabel" name="Puntos2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Puntos2</string>
   </property>
  </widget>
  <widget class="QLabel" name="Ganador">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ganador</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Reiniciar">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Reiniciar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P10">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P11">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>360</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P11</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P12">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>360</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P12</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P13">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>480</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P13</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P14">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>480</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P14</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P15">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>480</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P15</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P16">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>480</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P16</string>
   </property>
  </widget>
  <widget class="QLabel" name="C9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>303</y>
     <width>59</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>C9</string>
   </property>
  </widget>
  <widget class="QLabel" name="C10">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>303</y>
     <width>59</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>C10</string>
   </property>
  </widget>
  <widget class="QLabel" name="C11">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>303</y>
     <width>59</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>C11</string>
   </property>
  </widget>
  <widget class="QLabel" name="C12">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>303</y>
     <width>59</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>C12</string>
   </property>
  </widget>
  <widget class="QLabel" name="C13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>423</y>
     <width>59</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>C13</string>
   </property>
  </widget>
  <widget class="QLabel" name="C14">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>423</y>
     <width>59</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>C14</string>
   </property>
  </widget>
  <widget class="QLabel" name="C15">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>423</y>
     <width>59</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>C15</string>
   </property>
  </widget>
  <widget class="QLabel" name="C16">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>423</y>
     <width>59</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>C16</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P17">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P17</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P18">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>140</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P18</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P19">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>140</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P19</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P20">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>140</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P20</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P21">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P21</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P22">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>260</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P22</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P23">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>260</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P23</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P24">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>260</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P24</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P25">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>360</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P25</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P26">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>360</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P26</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P27">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>360</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P27</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P28">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>360</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P28</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P29">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>480</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P29</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P30">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>480</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P30</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P31">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>480</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P31</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P32">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>480</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>P32</string>
   </property>
  </widget>
  <widget class="QLabel" name="C28">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>330</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>C28</string>
   </property>
  </widget>
  <widget class="QLabel" name="C17">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>100</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>C17</string>
   </property>
  </widget>
  <widget class="QLabel" name="C18">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>100</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>C18</string>
   </property>
  </widget>
  <widget class="QLabel" name="C19">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>100</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>C19</string>
   </property>
  </widget>
  <widget class="QLabel" name="C20">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>100</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>C20</string>
   </property>
  </widget>
  <widget class="QLabel" name="C21">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>220</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>C21</string>
   </property>
  </widget>
  <widget class="QLabel" name="C22">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>220</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>C22</string>
   </property>
  </widget>
  <widget class="QLabel" name="C23">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>220</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>C23</string>
   </property>
  </widget>
  <widget class="QLabel" name="C24">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>220</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>C24</string>
   </property>
  </widget>
  <widget class="QLabel" name="C25">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>330</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>C25</string>
   </property>
  </widget>
  <widget class="QLabel" name="C26">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>330</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>C26</string>
   </property>
  </widget>
  <widget class="QLabel" name="C27">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>330</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>C27</string>
   </property>
  </widget>
  <widget class="QLabel" name="C29">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>440</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>C29</string>
   </property>
  </widget>
  <widget class="QLabel" name="C30">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>440</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>C30</string>
   </property>
  </widget>
  <widget class="QLabel" name="C31">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>440</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>C31</string>
   </property>
  </widget>
  <widget class="QLabel" name="C32">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>440</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>C32</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Salir">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
